docs/sources/panels-visualizations/visualizations/state-timeline/index.md

```diff
@@ -19,9 +19,97 @@ weight: 100
+A state timeline visualization displays data in a way that shows state changes over time. In a state timeline, the data is presented as a series of bars or bands called _state regions_. State regions can be rendered with or without values, and the region length indicates the duration or frequency of a state within a given time range.
 
-{{< figure src="/static/img/docs/v8/state_timeline_strings.png" max-width="1025px" caption="state timeline with string states" >}}
+For example, if you're monitoring the CPU usage of a server, you can use a state timeline to visualize the different states, such as “LOW,” “NORMAL,” “HIGH,” or “CRITICAL,” over time. Each state is represented by a different color and the lengths represent the duration of time that the server remained in that state:
+
+{{< figure src="/static/img/docs/state-timeline-panel/state-timeline-panel.png" max-width="1025px" alt="A state timeline panel showing CPU usage" >}}
+
+The state timeline visualization is useful when you need to monitor and analyze changes in states or statuses of various entities over time. You can use one when you need to:
+
+- Monitor the status of a server, application, or service to know when your infrastructure is experiencing issues over time.
+- Identify operational trends over time.
+- Spot any recurring issues with the health of your applications.
+
+## Configure a state timeline
+
+The following video shows you how to configure a state timeline:
+
+<!-- video TBA here -->
+
+1. Click the Grafana menu icon and, in the sidebar, click Dashboards.
+
+1. On the Dashboards page, click New in top right corner and select New Dashboard in the drop-down or select an existing dashboard.
+
+1. Click + Add visualization.
+
+1. Under the visualization type dropdown, change the visualization type from Time series to State timeline.
+
+1. In the Query tab below the graph, select your data source of choice. For example, if you have the [Grafana TestData data source](https://grafana.com/docs/grafana/latest/datasources/testdata/) configured, select CSV content from the scenario dropdown.
+
+1. Copy the following CSV data as an example:
+
+```
+Timestamps,Server A,Server B
+2024-02-29 8:00:00,Up,Up
+2024-02-29 8:15:00,null,Up
+2024-02-29 8:30:00,Down,null
+```
+
+1. Under Transform data tab, click + Add transformation and click [Convert field type](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/#convert-field-type).
+
+1. Convert the Timestamps field to be of type time.
+
+1. Add customization to your state timeline by updating the different [state timeline options](#state-timeline-options).
```

Comment:
Let's chat about this addition. We don't want to be repeating what is a fairly simple process (creating a visualization) over and over in the docs. We have docs that explain how to add visualizations that we can point folks to. Is the focus here the transformation?

Reply:
[Create a dashboard](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/) describes how to create a dashboard including adding a visualization and a data source.